### DISCUSSION TITLE
When the next version will be released ?

### DISCUSSION THREAD

**Discussion by GaylordP** [react-navigation/react-navigation#10762] [Q&A] [answered]
Hello and thank you for your library :)

Do you have an approximate release date for the next release?
Because I would like to be able to benefit [from this pull request](https://github.com/react-navigation/react-navigation/pull/10682) :)

Thanks you very much :)

**Comment by raajnadar** [comment, marked as answer]
Released in this version, you can use the API now https://github.com/react-navigation/react-navigation/releases/tag/%40react-navigation%2Fcore%406.3.0

**Comment by GaylordP** [comment]
Thanks very much :)
